lib/louisville/extensions/history.rb

```diff
@@ -27,8 +27,8 @@ def self.included(base)
           extend ::Louisville::Extensions::History::ClassMethods
 
           # If our slug has changed we should manage the history.
-          after_save :delete_matching_historical_slug,  :if => :louisville_slug_changed?
-          after_save :generate_historical_slug,         :if => :louisville_slug_changed?
+          after_save :delete_matching_historical_slug,  if: -> { ActiveRecord.version.to_s >= '5' ? louisville_slug_previously_changed? : louisville_slug_changed? }
```

Comment:
If you've changed the Gemfile to require ActiveRecord 5.2.X, are the checks for ActiveRecord.version necessary?

Reply:
yes, i'll do this.

Reply:
@aalong-tr I've updated it now.